portal/src/main/webapp/config_service.jsp

```diff
@@ -154,7 +154,9 @@
                 enableOncoKBandHotspots = "\"custom\"";
         }
              
-        obj.put("oncoprintOncoKbHotspotsDefault",enableOncoKBandHotspots);    
+        obj.put("oncoprintOncoKbHotspotsDefault",enableOncoKBandHotspots);
```

Comment:
strange that this line shows up as modified

Reply:
trailing spaces removed